lib/datadog/core/environment/process.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+require_relative '../normalizer'
+
+module Datadog
+  module Core
+    module Environment
+      # Retrieves process level information such that it can be attached to various payloads
+      module Process
+        module_function
+
+        # Returns the last segment of the working directory of the process
+        # @return [String] the last segment of the working directory
+        def entrypoint_workdir
+          File.basename(Dir.pwd)
+        end
+
+        # Returns the entrypoint type of the process
+        # @return [String] the type of the process, which is fixed in Ruby
+        def entrypoint_type
+          Core::Environment::Ext::PROCESS_TYPE
```

Comment:
We can remove `Core::` from this constant access (see comment in `def serialized`).

Reply:
Thanks for this note! see https://github.com/DataDog/dd-trace-rb/pull/5033/commits/31d97964fedc4d2f187f25e484ee87d9b3e2a2be!